feature/1396 - Add San Jose International Airport

JPmAn24 opened the pull request:
Resolves #1396 

The purpose of this pull request is to add San Jose Norman Y Mineta International Airport.

N864T commented:
FYI
- _SJC3OAK121 = Intercept point of the OAK 121 radial for the SJC3 SID
- _SUNOL1SJC009 = Intercept point of the SJC 009 radial for the 30 deps on the SUNOL1 SID	    	
- _090OAK120 = Heading 090 from 1.8 DME SJC --> OAK 120 radial

N864T commented on assets/airports/ksjc.json:
```diff
@@ -482,48 +482,74 @@
     		"speed": "",
     		"method": "random",
-    		"rate": null,
+    		"rate": 7,
```
Do these rates make sense? The yearly total ('19-'20) of IFR flights was 234,701. Divided by 8760 and divided amongst the spawnPatterns give ~5, but I bumped it up to 7.

N864T commented:
@JPmAn24 If you have the time could you create the terrain for SJC? I do most of my work on a Raspberry Pi and it struggles even with something as simple as GIMP. Thanks!

JPmAn24 commented:
@N864T I’ll get that done! Thanks for taking this over!

JPmAn24 commented:
Ok @N864T  Just a couple things to fix while I'm here. So far, the work you've done here is great!

I've added the fixes HNNLE MOONY and PXN
Added ksjc.geojson
Changed has_terrain to true
Corrected format of the videomap

Changes will be up in a few minutes!

JPmAn24 commented:
Here it is in all of it's glory!
![image](https://user-images.githubusercontent.com/28244253/82953756-f42d8980-9f70-11ea-9c73-2848928ddf1f.png)

N864T commented:
Thanks!

Fechulo reviewed (changes requested):
@N864T Great work on this so far! I've left some comments on the airport guide and airportLoadList as well as the comments below. Work your way through them and when you're ready you can request another review from me directly and I'll check all the boxes as I verify the changes. Practical review still pending. If you have any questions let me know :)

- [ ] line 2: AIRAC can be made 2009 at this point
- [ ] lines 5&6: Change to "Norcal", otherwise the voice synthesizer might spell it out
- [ ] line 28: As I commented on the airport guide. The airspace here seems to be some kind of combination of multiple areas of the NCT. Since we only care about SJC here, the airspace should be a combination of the Area A sectors, as described here: https://www.oakartcc.org/web/viewer.html?file=/controllers/file/7576a83b-5e65-11e9-8010-2a32edb55910
- [ ] line 145: Align colon

LOUPE5:
- [ ] _RWY12R1.8DME is a fly-over fix denoted with a `^`, so it should be `"^_RWY12R1.8DME"`
- [ ] the waypoint _090OAK120 should be renamed to "_OAK120019".
- [ ] The line from _12R1.8DME to _OAK120019 should be draw here as well

SJC3:
- [ ] _RWY12R1.8DME is a fly-over fix denoted with a `^`, so it should be `"^_RWY12R1.8DME"`
the waypoint _SJC3OAK121 seems to be slightly off from where it should be. The waypoint SPTNS seems to be in pretty much the right spot so feel free to reuse its coordinates. It should also be renamed to - [ ] "_OAK121021".
- [ ] The line from _12R1.8DME to _OAK121021 should be draw here as well

SUNOL1:
- [ ] _RWY30R5DME is a fly-over fix denoted with a `^`, so it should be `"^_RWY30R5DME"`. Also _SUNOL1turnleft should be renamed to be defined as a radial distance from SJC.
- [ ] _RWY12R1.8DME is a fly-over fix denoted with a `^`, so it should be `"^_RWY12R1.8DME"`
- [ ] Make sure you include these fixes in the draw section as well

TECKY3:
- [ ] STCLR and MLPTS should be fly-over fixes. the altitude restriction on them should be "A09+" since it's 900ft (i.e. FL09 not FL90)

STARs:
- [ ] Make sure STARs are defined in alphabetical order.

PYE3:
- [ ] Make sure entryPoints are in alphabetical order
- [ ] SAC transition should be `SAC` to `POPES` then to `PYE`
- [ ] `STINS` is missing in between `PYE` and `HADLY`
- [ ] There's no need for the headings after `OSI`

RISTI1:
- [ ] CEDES is a fly over fix

BRINY2:
- [ ] Missing waypoint `FATHM`

BRIXX2:
- [ ] YADUT is a fly over fix

FRLON2:
- [ ] PEGGS is a fly over fix

RAZRR4:
- [ ] Make sure the minimum altitudes come before the maximum altitudes, they're the wrong way around.
- [ ] ZORSA is a fly over fix
- [ ] The runway transitions are the wrong way around. arrivals to 30s should go to SEKKO and 12s should go to GOTEE

ROBIE5:
- [ ] Missing waypoint GOALI
- [ ] No need for the headings after GILRO or SJC

SILCN4:
- [ ] Missing waypoint TROXX
- [ ] Make sure the minimum altitudes come before the maximum altitudes, some of them are the wrong way around.
- [ ] ZORSA is a fly over fix
- [ ] Remove the extra spaces at the end of line 475

- [ ] Is there another video map we could use with more of an emphasis on KSJC. The lack of an extended centerline is far from ideal,
